pylint/checkers/base.py

```diff
@@ -1663,9 +1661,8 @@ class LambdaForComprehensionChecker(_BasicChecker):
                       'deprecated-lambda',
                       'Used when a lambda is the first argument to "map" or '
                       '"filter". It could be clearer as a list '
-                      'comprehension or generator expression.',
-                      {'maxversion': (3, 0)}),
-           }
+                      'comprehension or generator expression.')
```

Comment:
Ditto regarding removing the check altogether

Reply:
Shall I be removing this entire class?
https://github.com/99/pylint/blob/2bdbeadcc7717230f68b413a46572617afb53660/pylint/checkers/base.py#L1653

Reply:
Yup!